src/rotation-lock.cpp

```diff
@@ -205,6 +226,12 @@ class RotationLockIndicator::Impl
     g_menu_append_section(menu, NULL, G_MENU_MODEL(section));
     g_object_unref(section);
     g_object_unref(menu_item);
+    section = g_menu_new ();
+    menu_item = g_menu_item_new (_("Display settings…"), "indicator.settings");
+    g_menu_append_item (section, menu_item);
+    g_menu_append_section (menu, NULL, G_MENU_MODEL (section));
+    g_object_unref (section);
+    g_object_unref (menu_item);
```

Comment:
I am not sure how lomiri handles this in desktop convergent mode, but I sense we should add some if clause around the above that checks that we don't add the display settings menu item on Lomiri.